ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+token_dpu_operator_key=$(cat "/var/run/token/dpu-token/dpu-key")
+endpoint=$(cat "/var/run/token/dpu-token/url")
+
+job_url="https://${endpoint}/job/Lab144_DPU_Operator_Test/lastBuild"
+
+while : 
+do
+    job_check=$(curl -k -s --resolve "${endpoint}:443:10.0.180.88" "$job_url/api/json")
+    result=$(echo "$job_check" | jq -r '.result')
+    if [ $result != "null" ]; then
+	sleep $((RANDOM % 61 + 60))
+        break
+    else
+        sleep 60
```

Comment:
We can indefinitely sleep if there is a problem with the job. For the PROW CI workflow, can we bail out if the retries are too many? If so, we might want to add this here.

Reply:
great idea! I will add this in